src/Randomizer.SMZ3.Tracking/tracker.json

```diff
@@ -1660,25 +1781,36 @@
       "AskChatAboutContent": [
         "Hmm. I'm not so sure about that. Let's ask the professionals in chat if that was some hashtag content.",
         "Do you really think that was high quality hashtag content? Let's get some unbiased opinions from chat to confirm.",
-        "Error. My metrics calculation algorithm could not determine the quality of that hashtag content. Falling back to chat confirmation of content quality."
+        "Error. My metrics calculation algorithm could not determine the quality of that hashtag content. Falling back to chat confirmation of content quality.",
+        "You have been randomly selected to have your content screened. Please let the council know if this was content or not.",
+        "Viewers at home. Please use the new poll to let us know how good this content is, thank you.",
+        "You might think that was some good content but do the viewers agree? Let's find out!"
       ],
       "AskChatAboutContentYes": [
         "It's your lucky day. Chat has confirmed that was some hashtag content.",
+        "It looks like the chat and I are in agreement. That was some solid content.",
+        "I agree with the chat, that was some great content",
+        "Looks like the chat is on your side. For now.",
         "I can't say that I agree with the results, but the chat indeed believes that was some hashtag content.",
         "Congratulations! You're the lucky winner of our grand prize."
       ],
       "AskChatAboutContentNo": [
         "I'm glad I asked. The chat has denied your request to increase your content levels.",
+        "I am sorry but the chat does agree that what happened was good content. Please try again later",
+        "Seriously chat? I thought that was good content but the rules are rules.",
```

Comment:
Again, I'm not too sure about these.

That said, a more "positive" tracker as a whole could be funny sometimes. But that would require profiles or some other system to keep track of her mood in the configs.

Reply:
My intent with both was to cause friction between both the streamer and the chat with tracker being the instigator

Reply:
Hmm, I suppose that's fair.